Orion.Net.Scripts.Common/Diagnostics/SendImageContentClientScript.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Orion.Net.Client.Configuration;
+using Orion.Net.Client.Scripts;
+using Orion.Net.Core.Scripts;
+
+namespace Orion.Net.Scripts.Common.Diagnostics
+{
+    /// <summary>
+    /// Take a screenshot on the client computer
+    /// </summary>
+    public class SendImageContentClientScript : BaseClientScript
+    {
+        public SendImageContentClientScript(Connector connector)
+            : base(connector)
+        {
+            identifier = Guid.NewGuid();
+            AvailableParameters.Add(new ScriptParameter()
+            {
+                Name = "filePath"
```

Comment:
You can create a private constant for this parameter name, this will avoid repeating hard coded string